tac/project-updates/2025/2025-Q2-Cacti.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: 2025 Q2 Cacti
+parent: 2025
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- We are forging ahead with the IETF SATP reference implementations, the codebase integration and a mentorship project proposal which was granted acceptance just a few days ago.
+- We are still working on the Cacti Chainlink CCIP connector which we've presented in the February LFDT Interoperability Workshop mid-February this year.
+- We had an LFDT mentorship project approved for the summer, which will further the goal of creating an integrated architecture as envisioned in the project's roadmap.
+- A steady stream of questions are being answered on Discord where most of our activity is happening these days.
+- There haven't been any violations of the code of conduct by contributors so we count ourselves lucky so far.
+- There's been a bit of a slowdown this quarter, the number of active Contributors are decreased by 9.09%.
+
+# Questions/Issues for the TAC
+
+N/A
+
+# Releases
+
+December 2, 2024 - `v2.1.0`
+
+We've had plans to issue v2.2.0 in this quarter but there is a chain of dependencies tied in a bit of a gordian knot with the release process, the package renaming and the `satp-dev` branch which contains most of the work of the IETF SATP team. We are hoping to have the new release issued within a few weeks from now when the `satp-dev` branch is properly merged onto `main`.
+
+# Overall Activity in the Past Quarter
+
+We regularly respond to questions! 
+With that said the mailing list is pretty quiet. 
+The Discord channels and DMs are where most of the communication happens.
+
+New features are being developed and we have at least 3 active maintaniers who can step in when needed.
+
+Technical changes that we are working on:
+1. Get release process fully migrated to the new GitHub organization
+2. Better package.json validation across the board with the declarative constraint validation feature of a library called `joi` (we have 60+ packages)
+3. Bringing down the Dependabot alerts
+4. Increasing our OpenSSF score, particularly with the low hanging fruit such as not committing binaries into the git repository
+   
+# Current Plans
+
+We'll want to do a major release this summer because we have to have some breaking changes.
+We need to migrate away from Web3JS which was unfortunately deprecated and retired as of last month (March 2025).
```

Comment:
I've had a short panic attack reading this as we've just recently approved the web3j annual report. A careful reader will spot that Web3JS != `LFDT-web3j/web3j`. 🤓